package.json

```
@@ -8,24 +8,9 @@
     "build": "tsc && rsbuild build",
     "start": "rsbuild dev",
     "test": "vitest",
-    "lint": "eslint 'src/**/*.{ts,tsx}'",
-    "lint-staged": "lint-staged --config lint-staged.cjs",
-    "preview": "rsbuild preview",
-    "format": "yarn run prettier --write 'src/**/*.{ts,tsx}'",
-    "prepare": "husky install"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest",
-      "prettier"
-    ],
-    "plugins": [
-      "prettier"
-    ],
-    "rules": {
-      "prettier/prettier": "error"
-    }
+    "format": "biome format ./src --write",
+    "lint": "biome lint ./src",
```

@ap-justin We'll want to make sure all the package.json script commands are sync'd across repos:
```suggestion
    "lint": "biome lint ./src",
    "check": "biome check --apply src",
    "check-unsafe": "biome check --apply-unsafe src"
```